bin/test.py

```diff
@@ -79,6 +84,22 @@ def create_bucket(client, name, region):
     else:
         client.create_bucket(Bucket=name, ACL='public-read', CreateBucketConfiguration={ 'LocationConstraint': region })
 
+def empty_cfn_bucket(session, config, stack_id):
+    """ Empty codepipeline bucket before fargate stack deletion as the bucket 
+        isnt empty so stack deletion fails.
+    """
+    cfn_client =  session.client('cloudformation', region_name='us-east-1', config=config)
+    cfn_stack = cfn_client.describe_stacks(StackName=stack_id)['Stacks'][0]
+    fargate_stack = None
+    for output in cfn_stack['Outputs']:
+        if output['OutputKey'] == 'FargateStackName':
+            fargate_stack = output
+    s3_bucket_name = cfn_client.describe_stack_resource(StackName=fargate_stack['OutputValue'],LogicalResourceId='CodePipelineArtifactBucket')['StackResourceDetail']['PhysicalResourceId']
+    s3_client = session.client('s3', region_name='us-east-1', config=config)
+    all_objs = s3_client.list_objects_v2(Bucket=s3_bucket_name)
```

Comment:
Does this work to delete all objects in the bucket?

bucket = boto3.resource('s3').Bucket('bucketname')
bucket.objects.all().delete()
bucket.delete()